### ISSUE TITLE
Issue with live-reloading using cordova-android 7.0.0

### ISSUE THREAD

**Issue by wytlytningNZ** [isleofcode/corber#435]
Steps to reproduce:
corber: `1.1.10`
cordova-android: `7.0.0`

1. Install android 7.0.0 `<engine name="android" spec="7.0.0" />`
2. `corber prepare`
3. `corber s --environment=development --platform=android`

Errors with this message:
```
WARNING: corber 
Did not find cordova.js or cordova_plugins.js at platforms/android/assets/www. Ember App LiveReload will still work, but device & plugin APIS will fail.
```

I believe this is because cordova-android nukes the `www` folder after doing a prepare (which removes `cordova.js`, `cordova-plugins.js` and `plugins/*`. [See here:]( https://github.com/apache/cordova-android/blob/d8d92cae61ca6049f8590adced3f4dfce3892a94/bin/lib/create.js#L67)

I managed to fix this locally [by modifying](https://github.com/isleofcode/corber/blob/6f8756a8ca929a884f31e97b3118a516f9028cbf/lib/targets/cordova/utils/cordova-assets.js#L13) `getPaths` to check if `www` folder exists, otherwise setting the asset path to `android/platform_www`.
```
if (platform === 'ios') {
      assetsPath = path.join(platformsPath, 'ios', 'www');
    } else if (platform === 'android') {
      assetsPath = path.join(platformsPath, 'android', 'assets', 'www');
      // cordova-android 7.0 no longer adds provides assets/www
      // path is now android/platform_www
      if (!existsSync(assetsPath)) {
        assetsPath = path.join(platformsPath, 'android', 'platform_www');
      }
    } else if (platform === 'browser') {
      assetsPath = path.join(platformsPath, 'browser', 'www');
    }
```

What are your thoughts?

**Comment by alexblom** [comment]
Thanks for the report. I've replicated and agree this is the best solution for now. I'm happy to work this into a PR, but will leave it open a few days in case you'd like to contribute it in first :)

**Comment by wytlytningNZ** [comment]
I will submit a PR today hopefully 🤞

**Comment by wytlytningNZ** [comment]
Currently blocked by this issue:
https://github.com/isleofcode/corber/issues/444

**Comment by alexblom** [comment]
Released in 1.1.14 :) Thanks for the PR.
